core/src/script/CGXP/tools/autoProject.js

```diff
@@ -0,0 +1,91 @@
+/**
```

Comment:
But why name it ``OpenLayers.AutoProjection``? Isn't it some CGXP specific code?